test-app/runtime/build.gradle

```diff
@@ -130,20 +140,49 @@ android {
         }
     }
 
+    publishing {
+        singleVariant('release') {
+            withSourcesJar()
+        }
+    }
+
 }
 
-allprojects {
-    gradle.projectsEvaluated {
-        tasks.withType(JavaCompile) {
-            options.compilerArgs << "-Xlint:all" << "-Werror"
+publishing {
+    publications {
+        release(MavenPublication) {
+            groupId = 'org.nativescript.android'
+            artifactId = 'runtime'
+            version = '1.0-SNAPSHOT'
+
+            afterEvaluate {
+                from components.release
+            }
         }
     }
+    repositories {
+        mavenLocal()
+    }
 }
 
 dependencies {
-    // println "\t ~ [DEBUG][runtime] build.gradle - ns_default_junit_version = ${ns_default_junit_version}..."
+    def androidXAppCompatVersion = "${ns_default_androidx_appcompat_version}"
+    if (project.hasProperty("androidXAppCompat")) {
+        androidXAppCompatVersion = androidXAppCompat
+        outLogger.withStyle(Style.SuccessHeader).println "\t + using android X library androidx.appcompat:appcompat:$androidXAppCompatVersion"
+    }
+
+    def androidXMaterialVersion = "${ns_default_androidx_material_version}"
+    if (project.hasProperty("androidXMaterial")) {
+        androidXMaterialVersion = androidXMaterial
+        outLogger.withStyle(Style.SuccessHeader).println "\t + using android X library com.google.android.material:material:$androidXMaterialVersion"
+    }
 
     implementation fileTree(include: ['*.jar'], dir: 'libs')
+    implementation "com.google.android.material:material:$androidXMaterialVersion"
+    implementation "androidx.appcompat:appcompat:$androidXAppCompatVersion"
+    implementation 'androidx.navigation:navigation-fragment-ktx:2.5.3'
+    implementation 'androidx.navigation:navigation-ui-ktx:2.5.3'
```

Comment:
are these needed? It'd mean all apps now depend on material, androidx navigation and appcompat